### PR TITLE
Updated dynamic PEP explanation involving BytesBuf (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: docs/plugins/dynamic_policy_enforcement_points.mdpp [hunk 1]
```diff
@@ -112,7 +113,7 @@ becomes:
 
     pep_api_data_obj_put_pre(*INSTANCE_NAME, *COMM, *DATAOBJINP, *BUFFER, *PORTAL_OPR_OUT)
 
-After `*INSTANCE_NAME`, note the next two types of this signature are serialized by the framework, but the last two are not.  This means that the information within `*COMM` and `*DATAOBJINP` will be available in the rule logic whereas `*BUFFER` and `*PORTAL_OPR_OUT` will explain they are not supported.
+After `*INSTANCE_NAME`, note the next three types of this signature are serialized by the framework, but the last one is not.  This means that the information within `*COMM`, `*DATAOBJINP`, and `*BUFFER` will be available in the rule logic whereas `*PORTAL_OPR_OUT` will explain they are not supported.
```
Comment (trel) [inline docs/plugins/dynamic_policy_enforcement_points.mdpp:116]: "explain they are not" -> "explain it is not"

Comment (korydraughn) [reply docs/plugins/dynamic_policy_enforcement_points.mdpp]: Done.

**Review by alanking — APPROVED** [review]

**Comment by korydraughn** [comment]
#'d

**Comment by alanking** [comment]
Is this needed in 4-2-stable as well?

**Pull request merged**